manage/sawtooth_manage/docker.py

```diff
@@ -216,7 +216,7 @@ def _built_in_processor_types(self):
 
     def stop(self, node_name):
         state_file_path = os.path.join(self._state_dir, 'state.yaml')
-        state = yaml.load(file(state_file_path))
+        state = yaml.load(open(state_file_path))
```

Comment:
looks like a potential filehandle leak in handling of open here; open should almost always be done:

```
with open(...) as fd:
     s = yaml.load(fd)
```